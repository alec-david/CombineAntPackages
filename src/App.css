.App {
  text-align: center;
}

.App-intro {
  font-size: large;
}

.dropzone {
  display: table;
  margin: 0 auto;
}
