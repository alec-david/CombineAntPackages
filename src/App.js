import React, { Component } from "react";
import "./App.css";

import AddAndCombinePackages from "./components/AddAndCombinePackages";

class App extends Component {
  render() {
    return (
      <div className="App">
        <h1>Combine Ant Packages</h1>
        <AddAndCombinePackages />
      </div>
    );
  }
}

export default App;
