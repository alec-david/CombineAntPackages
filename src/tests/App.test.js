import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";

import App from "../App";

describe("Component: App", () => {
  it("should match its empty snapshot", () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
