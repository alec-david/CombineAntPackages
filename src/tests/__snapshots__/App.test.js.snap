// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should match its empty snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="App"
  >
    <h1>
      Combine Ant Packages
    </h1>
    <AddPackage />
  </div>,
  "nodes": Array [
    <div
      className="App"
    >
      <h1>
        Combine Ant Packages
      </h1>
      <AddPackage />
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <App />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": App {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="App"
        >
          <h1>
            Combine Ant Packages
          </h1>
          <AddPackage />
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="App"
        >
          <h1>
            Combine Ant Packages
          </h1>
          <AddPackage />
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <App />,
}
`;
